src/main/java/core/basesyntax/TreeSetCharacters.java

```diff
@@ -17,7 +22,31 @@
  * Результат 2: acf</p>
  */
 public class TreeSetCharacters {
-    public String getUniqueCharacters(String fileName) {
-        return null;
+    private static final int LETTERS_AMOUNT = 5;
+
+    public String getUniqueCharacters(String fileName) throws IOException {
+        TreeSet<Character> lettersFromFile = new TreeSet<>();
+        try (BufferedReader reader = new BufferedReader(new FileReader(fileName))) {
+            int input = reader.read();
+            while (input != -1) {
+                if (input >= 'A' && input <= 'Z') {
+                    input += 32;
```

Comment:
if input is char, than you can use char.toLowerCase (google method name :) ) instead of adding magic number 32.

Reply:
i thought that method was only in String/StringBuffer.